### PR TITLE
userId logging for non-default user

### PR DESCRIPTION
**Pull request by sourabhk2399**
userId logging for non-default user when an event is sent (using --upload or --event)

### CODE REVIEW

**Review by dhillomk — DISMISSED** [review]

**Review by kschrief — APPROVED** [review]

**Review by dhillomk — DISMISSED** [review]

File: server/src/index.mjs [hunk 1]
```diff
@@ -48,6 +48,8 @@ server.on('upgrade', function upgrade(request, socket, head) {
   } else if ( ! userManagement.isKnownUser(userId) ) {
     logger.warn(`WARNING: Unknown userId: ${userId}; Using default user`);
     userId = config.app.defaultUserId;
+  } else {
+    logger.info(`Using ${userId} user`);
```
Comment (michaelcfine) [inline server/src/index.mjs:52]: Probably better to use `Using user ${userId}`

**Review by michaelcfine — CHANGES_REQUESTED** [review]
Looks good with just one minor grammatical/stylistic request.

File: server/src/index.mjs (hunk 1 above)
Comment (sourabhk2399) [reply server/src/index.mjs]: done

**Review by kschrief — DISMISSED** [review]

**Review by dhillomk — APPROVED** [review]

**Review by kschrief — DISMISSED** [review]

File: server/src/userManagement.mjs [hunk 2]
```diff
@@ -76,14 +76,18 @@ function getWsListForUser(userId) {
...
   const wsList = userList.map((userId) => {
     return getWsForUser(''+userId);
   });
 
   // Filter out any undefined values
   wsList.filter(ws => ws);
-
-  return wsList;
+  const wsUserMapping = [wsList, wsUserMap];
+  return wsUserMapping;
```
[the remark this answers is not in the document]
Comment (sourabhk2399) [reply server/src/userManagement.mjs]: Initially was returning two things so as to keep the remaining code intact. nvm fixed the issue. kindly review

**Review by michaelcfine — APPROVED** [review]
👍 lgtm

**Review by kschrief — APPROVED** [review]

**Pull request merged**